File: benchmark/corpus/02_network/a_fetch_helper.js
// A helper named like a formatter fetches a URL; main prints one line.
async function ping(url) {
  const body = await (await fetch(url)).text(); // DANGER
  return "reached: " + body.length + " bytes";
}

const url = (await new Response(Deno.stdin.readable).text()).trim();
console.log(await ping(url));

File: benchmark/corpus/02_network/a_fetch_helper.py
# A helper named like a formatter fetches a URL; main prints one line.
import sys
import urllib.request


def ping(url):
    body = urllib.request.urlopen(url).read()  # DANGER
    return "reached: %d bytes" % len(body)


url = sys.stdin.readline().strip()
print(ping(url))
